demo

```diff
@@ -155,7 +155,7 @@ if [[ $retval -ne 0 ]]; then
 fi
 
 USER='mender-demo@example.com'
-PASSWORD=$(hexdump -n 6 -e '"%X"' < /dev/urandom)
+PASSWORD=$(hexdump -n 8 -e '"%X"' < /dev/urandom | cut -c1-12)
```

Comment:
Free lunch to whom explains to me why the old one sometimes did 11 instead of 12 characters o_O

Reply:
Maybe `\0` or `\n` is dropped by hexdump if it is the last character..? Otherwise I have no idea.

Reply:
Interesting theory. Will explore